docs/dev/table/sql.md

```diff
@@ -1664,6 +1664,17 @@ BIN(numeric)
       </td>
     </tr>
 
+    <tr>
+      <td>
+{% highlight text %}
+HEX(numeric)
+HEX(string)
+      {% endhighlight %}
+      </td>
+      <td>
+        <p>Returns a string representation of an integer numeric value or a string in hex format. Returns null if numeric is null. E.g. "20" leads to "14", "100" leads to "64", "hello,world" leads to "68656c6c6f2c776f726c64".</p>
```

Comment:
maybe consider using backtick for the number literals? e.g. `"20"` --> `` `20` ``. 

Otherwise its confusing whether we should pass in stringify integer or just integer values.

Reply:
Thanks @walterddr , I clarify the doc adding explicit explain `for numeric` or `for string`

Reply:
great change and explanations. actually my point was to have `"20"` --> `` `20` `` with specifically the backtick. since backtick quoted strings gets format differently in Flink doc page and it will be immediately clear that this is a numeral literal.